app/models/ingest_job.rb

```diff
@@ -23,6 +23,8 @@ class IngestJob
   attr_accessor :reparse
   # Boolean indication of whether or not to delete file from bucket on parse failure
   attr_accessor :persist_on_fail
+  # extra key/value storage of attributes
+  attr_accessor :options
```

Comment:
I know this is just a draft, but I started reviewing it without realizing that first :).  Instead of creating a generic 'options' hash, consider creating a class `differential_expression_params` and including one here (sort of similar to how we embed `expression_file_info` inside StudyFile.  That way all the documentation, parsing, and validation logic for those params can live in a more easily findable place